Lib/test/test_unicode.py

```diff
@@ -78,7 +83,7 @@ def test_literals(self):
         self.assertEqual('\uffff', '\U0000ffff')
         self.assertRaises(SyntaxError, eval, '\'\\Ufffffffe\'')
         self.assertRaises(SyntaxError, eval, '\'\\Uffffffff\'')
-        self.assertRaises(SyntaxError, eval, '\'\\U%08x\'' % 0x110000)
+        self.assertRaises(SyntaxError, eval, '\'\\U%08x\'' % INVALID_CODEPOINT)
```

Comment:
It is important that it is not just some invalid code, but the first code past the valid range (`sys.maxunicode + 1`).